### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sjparkinson**
Work through most of the TODOs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,43 +4,45 @@ A GitHub App that helps maintain a large number of repositories in a GitHub orga
...
+## Development
+
+### Getting Started
+
+1. Build the application
 
 ```sh
-# Install dependencies
+# Install the correct version of Node.js (defined in .nvmrc)
+nvm use
```
Comment (simonplend) [inline README.md]: Oh no! What if I'm not using `nvm`?! :scream:

File: README.md [hunk 2]
```diff
@@ -4,43 +4,45 @@ A GitHub App that helps maintain a large number of repositories in a GitHub orga
...
 
-```sh
-# If you use nvm, set the version of node to use (defined in our .nvmrc)
-# https://github.com/creationix/nvm#nvmrc
-nvm use
+4. Install the app in your personal GitHub account, then pick an example repository or two in the drop down
 
-# For handy debug log messages when running the app
-echo -e "\nLOG_LEVEL=trace" >> .env
+5. Probot will then automatically create you a `.env` file (you can add `LOG_LEVEL=trace` for more verbose logging)
```
Comment (simonplend) [inline README.md]: Where is this `.env` file?

File: README.md [hunk 3]
```diff
@@ -4,43 +4,45 @@ A GitHub App that helps maintain a large number of repositories in a GitHub orga
...
 
-# Run the app with nodemon and restart on code changes
-npm run dev
+6. From your browser, copy the installiation ID from the URL (e.g. at `https://github.com/settings/installations/123456` `123456` is the ID)
```
Comment (simonplend) [inline README.md]: Can you make the example

File: README.md (hunk 1 above)
Comment (sjparkinson) [reply README.md]: As in I should list other tools? Or?

Comment (simonplend) [reply README.md]: I was thinking it should say something along the lines of "if you use nvm", as it currently implies that you _need_ to be using `npm`.

**Pull request merged**